flexget/tests/test_qualities.py

```diff
@@ -109,6 +111,23 @@ def test_quality_failures(self, parser, test_quality):
         ))
 
 
+class TestQualityInternalParser(object):
+    @pytest.mark.parametrize("test_quality", [
```

Comment:
you can set it a two vars instead of one that'll create a tuple, might be a little easier to read imo